src/pages/admin/RequestManagementPage.jsx

```
@@ -51,41 +51,60 @@ const RequestManagementPage = () => {
   const [visibleCount, setVisibleCount] = useState(PAGE_BATCH_SIZE);
   const sentinelRef = useRef(null);
   const [alert, setAlert] = useState(null);
-  const [processingRequestId, setProcessingRequestId] = useState(null);
-  const [processingUsername, setProcessingUsername] = useState(null);
-  const [provisioningStatus, setProvisioningStatus] = useState(null);
+  // 백엔드 podCreationSemaphore가 최대 3건 동시 처리를 허용하므로, 프론트도 신청서별로
+  // 독립적으로 처리 상태를 추적한다 — 하나가 처리 중이어도 다른 신청서는 막히면 안 된다.
+  const [processingRequestIds, setProcessingRequestIds] = useState(() => new Set());
```

ESLint

[error] 'setProcessingRequestIds' is assigned a value but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)